### PR TITLE
Rename redemption code to coupon ID

### PR DESCRIPTION
**Pull request by slavcodev**

### CODE REVIEW

**Review by colinbird — DISMISSED** [review]

**Comment by slavcodev** [comment]
@Rebilly/php please review

**Comment by slavcodev** [comment]
@Rebilly/php please review

File: src/Entities/Coupons/Coupon.php [hunk 1]
```diff
@@ -12,29 +12,21 @@
...
+use Rebilly\Rest\Entity;
 
 /**
  * Class Coupon.
  */
-final class Coupon extends Resource
+final class Coupon extends Entity
 {
     /**
+     * @deprecated use {@see getId() instead}
```
Comment (ashkarpetin) [inline src/Entities/Coupons/Coupon.php:23]: Is this correct?

Comment (ashkarpetin) [inline src/Entities/Coupons/Coupon.php:23]:
```suggestion
     * @deprecated use {@see getId()} instead
```
?

File: src/Entities/Coupons/Redemption.php [hunk 2]
```diff
@@ -22,19 +22,41 @@ final class Redemption extends Entity
...
+     *
+     * @return $this
+     */
+    public function setCouponId($value)
+    {
+        return $this->setAttribute('couponId', $value);
+    }
+
+    /**
+     * @deprecated use {@see getCouponId() instead}
```
Comment (ashkarpetin) [inline src/Entities/Coupons/Redemption.php:41]:
```suggestion
     * @deprecated use {@see getCouponId()} instead
```
?

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**